src/libraries/System.Text.Json/src/System/Text/Json/JsonPropertyDictionaryJsonPropertyValueList.cs

```diff
@@ -0,0 +1,206 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+using System.Text.Json.Serialization.Metadata;
+
+namespace System.Text.Json
+{
+    internal sealed class JsonPropertyDictionaryJsonPropertyValueList : IList<JsonPropertyInfo>
```

Comment:
That class name is a bit of a mouthful, could it be shortened perhaps? Even though it's internal it would still appear in the debugger and stacktraces.

Reply:
I will shorten to `JsonPropertyDictionaryValueList`